hibernate-core/src/main/java/org/hibernate/sql/ast/SqlAstNodeRenderingMode.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+package org.hibernate.sql.ast;
+
+/**
+ * @author Christian Beikov
+ */
+public enum SqlAstNodeRenderingMode {
```

Comment:
Would appreciate a bit more Javadoc here, explaining why we need it, what functionality it relates to, etc.